src/src/Helpers/ESPEasy_FactoryDefault.cpp

```diff
@@ -216,7 +216,7 @@ void ResetFactory()
    */
   Settings.I2C_clockSpeed = DEFAULT_I2C_CLOCK_SPEED;
 
-  Settings.JSONBoolWithQuotes(DEFAULT_JSON_USE_QUOTES);
+  Settings.JSONBoolWithQuotes(DEFAULT_JSON_BOOL_USE_NO_QUOTES);
```

Comment:
This looks confusing.
The default suggests it is a setting to NOT use quotes, but the function name implies WITH quotes.

Reply:
Ah, inversion confusion, will fix